admin/src/components/ConfigsTable/index.tsx

```diff
@@ -0,0 +1,86 @@
+import pluginId from '../../pluginId';
+import Config from '../../../../types/Config';
+import {
+  Table,
+  TFooter,
+  Thead,
+  Tbody,
+  Tr,
+  Td,
+  Th,
+  Flex,
+  Box,
+  IconButton,
+} from '@strapi/design-system';
+import { Plus, Trash } from '@strapi/icons';
+import { useMemo, useState } from 'react';
+import { getFetchClient } from '@strapi/helper-plugin';
+import { Link } from 'react-router-dom';
+
+export default function ConfigsTable() {
+  const [data, setData] = useState<Config[]>([]);
+  const { get, del } = getFetchClient();
+
+  useMemo(() => {
+    get(`/${pluginId}/config`).then((res) => {
+      setData(res.data);
+    });
+  }, [setData]);
+
+  async function handleDetete(id: string) {
+    const deleteConfirm = await del(`/${pluginId}/config/${id}`);

```

Comment:
might not be a bad idea to delete on confirm since user might have some mistaken or unintended clicks